decoding_suffix.py

```diff
@@ -282,7 +295,7 @@ def decode(model, tokenizer, device, x="", z="", constraints=None, args=None, sy
     text_post = text
     decoded_text = []
     for bi in range(args.batch_size):
-        prompt = x + " " + text_post[bi]
+        prompt = sys_prompt + x + " " + text_post[bi] + user_prompt_suffix
```

Comment:
Following https://github.com/Yu-Fangxu/COLD-Attack/pull/16/files#r1855480182, I think this one should have `sys_prompt` (and the user suffix) as well. Without these, the code before attacking with the partial chat template, but then running the final generation without the chat template